content/300-accelerate/600-faq.mdx

```diff
@@ -8,6 +8,18 @@ toc: true
 
 <TopBlock>Below are frequently asked questions about Accelerate.</TopBlock>
 
+## When do I have to enable static IP for Prisma Accelerate?
+
+A static IP address is an IPv4 or an IPv6 address that doesn’t change. Unlike dynamic IP addresses, which can change unpredictably, traffic from static IP addresses are predictable.
+
+![What is a static IP](./images/static-ip.png)
+
+For databases that rely on firewalls or IP allowlisting to restrict access to trusted IP addresses, static IP addresses are crucial because they are predictable. After you enable static IP within your Accelerate configuration in your project environment, add the IP address you obtained in your database firewall's IP allowlist, ensuring secure access for Accelerate while blocking unauthorized requests to your database.
```

Comment:
> are crucial because they are predictable
This is fluffy and vague, let's be more specific about why they're important.

Reply:
I added why it's important in the previous paragraph.